vuepress/docs/v7.0/docs/getting-started/README.md

```diff
@@ -41,21 +29,20 @@ https://multipass.run/#install
 curl -sfL https://get.entando.org | bash
 ```
 
-3. The progress of the install will be displayed on the console. Installation can take 10+ minutes depending on how long the Docker images take to download. The sequence of steps performed by the CLI is identical to the manual steps below. To understand what the CLI does, review the manual steps.
+3. The progress of the install will be displayed on the console. Installation can take up to 10 minutes, depending on how long the Docker images take to download. The sequence of steps performed by the CLI is identical to the manual steps below. To understand what the CLI does, review the manual steps.
 4. The URL to access the Entando App Builder will print to the console once the install completes.
 5. Login with username:`admin` and password: `adminadmin`. Refer to [Login to Entando](#login-to-entando) for more information and next steps.
 
 ## Manual Install
 
-Manual installation allows you to configure a shared cluster or customize a local developer environment. The following learn-as-you-go approach gives you a working knowledge of Kubernetes as you get Entando up and running in a local environment. This is a three stage process:
+Manual installation allows you to configure a shared cluster or customize a local developer environment. The following learn-as-you-go approach gives you a working knowledge of Kubernetes as you get Entando up and running in a local environment. After downloading the necessary files, your instance of Kubernetes will be up and running in <60 seconds.
+
+This is a three stage process:
 
 1. [Install Kubernetes](#install-kubernetes)
 2. [Prepare the Kubernetes Environment](#prepare-kubernetes)
 3. [Deploy Entando](#deploy-entando)
 
-
-We've tested a variety of Kubernetes implementations, including Minikube, Minishift, CodeReady Containers, K3s, and Microk8s, to find the optimal combination of low CPU/memory usage, fast startup times, and minimal configuration. After downloading the necessary files, we'll have our own instance of Kubernetes up and running in <60 seconds.
```

Comment:
@nshaw, Not sure if removing this was a good idea, just thinking that a tutorial should keep peripheral information to a minimum.

Reply:
@jyunmitch i had the same thoughts here; i thought i'd changed it to read "we selected k3s to use the optimal combination..."
i don't think we need to dive into process and trial and error, just discuss what's topical and pertinent

Reply:
@jyunmitch looks good to me.